Assets/Scripts/Blocks/Block.cs

```diff
@@ -54,10 +55,10 @@ protected virtual void DestroyBlock()
     public void StartBlockDestructionCoroutine()
     {
         // Start destruction coroutine
-        StartCoroutine(StartBlockDestruction());
+        blockDestructionCoroutine = StartCoroutine(StartBlockDestruction());
     }
 
-    public void StartInstantBlockDestruction()
+    public void StartInstantBlockDestruction()//Why do we have this method?
```

Comment:
This replaces the original destruction coroutine when force outbreak destroys the block, as it's meant to be destroyed instantly